src/GitLabApiClient/Models/RateLimitPagedInfo.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace GitLabApiClient.Models;
+
+public class RateLimitPagedInfo
+{
+    public int NextPage { get; set; }
+    public int Page { get; set; }
+    public int PerPage { get; set; }
+    public int PrevPage { get; set; }
+    public int Total { get; set; }
+    public int TotalPages { get; set; }
+
+    public int RateLimitObserved { get; set; }
+    public int RateLimitRemaining { get; set; }
+    public DateTime RateLimitResetTime { get; set; }
+    public int RateLimitLimit { get; set; }
```

Comment:
I suggest to make all types nullable, if the header doesn't exist it would have the default value (for int its 0) and so you don't know if there are no pages or just the header didn't exist, etc

Reply:
Updated